pkg/chunkserver/pool.go

```diff
@@ -168,7 +168,7 @@ func (c *Cluster) createLogicalPool(logicalPool string) (LogicalPool, []Server)
 	}
 
 	// copysets
-	copysets = (int)(copysets / DEFAULT_REPLICAS_PER_COPYSET)
+	copysets = copysets / DEFAULT_REPLICAS_PER_COPYSET
```

Comment:
it should reserve

Reply:
Why? All parts are int.

Reply:
copysets can be set by user and it can may be float.

Reply:
I don't get it. `copysets` is declared in L121 as `copysets := 0`, how can it be set by users?

Reply:
line from 128 to 131 is to deal copysets, copysets variable is to record total value and copysetsPerChunkserver is defined by user but i comment it here for debug.

Reply:
`copysets` is int by definition and `copysetsPerChunkserver` may be a float. If so, in L165 `copysets += copysetsPerChunkserver` you need to convert `copysetsPerChunkserver` to int first and then add it to `copysets`. So `copysets` is always an integer.

Unless you change `copysets := 0` to `copysets := 0.0`.

Reply:
right, I ignored golang' division feature.